Helper/Data.php

```diff
@@ -228,7 +293,7 @@ public function canAllowMultiPleCurrency()
      */
     public function allowCurrencySpeicifcCountry()
```

Comment:
you could comment this spelling error also the way you did for getDisaplyBade, if that would be helpful for easily finding these typos later

Reply:
Just added a comment indicating the spelling error in the latest commit https://github.com/withreach/magento-plugin/commit/7839c5d4d81a2bda6427cd0af37885e7b62b9a48 . Thanks.